templates/workspace_services/databricks/terraform/network.tf

```diff
@@ -0,0 +1,165 @@
+resource "azurerm_network_security_group" "nsg" {
+  name                = local.network_security_group_name
+  location            = data.azurerm_resource_group.rg.location
+  resource_group_name = data.azurerm_resource_group.rg.name
+
+  tags = local.tre_workspace_service_tags
+
+  lifecycle { ignore_changes = [tags] }
+
+  security_rule {
+    name                       = "nsgsg-${local.workspace_resource_name_suffix}-001"
```

Comment:
is `nsgsg` what we use elsewhere or backed my a regular azure convention?
what does it mean?